README.md

```diff
@@ -27,6 +27,8 @@ A Flask application run in AWS Lambda used to process invoices from Alma. Invoic
 ALMA_SAP_INVOICES_ECR_IMAGE_NAME=### The name of the ECR image for the Alma SAP Invoices CLI.
 ALMA_SAP_INVOICES_ECS_CLUSTER=### The full ARN of the ECS cluster to run the Alma SAP Invoices CLI ECS task.
 ALMA_SAP_INVOICES_ECS_TASK_DEFINITION=### The family and revision (formatted as <family>:<revision>) or full ARN of Alma SAP Invoices CLI ECS task. 
+ALMA_SAP_INVOICES_ECS_GROUPS=### Security group(s) for the Alma SAP Invoices ECS task (formatted as a comma-separated string).
+ALMA_SAP_INVOICES_ECS_SUBNETS=### Subnet(s) for the Alma SAP Invoices ECS task (formatted as a comma-separated string).
```

Comment:
Related to another comment: I think this suffices.  We use this comma-separated env var type in a few projects, and I think it's common knowledge that you should not include spaces.

Reply:
See response in https://github.com/MITLibraries/alma-sapinvoices-ui/pull/83#discussion_r1744067445.